mobile/native/src/trade/position/handler.rs

```diff
@@ -138,7 +183,7 @@ pub fn update_position_after_dlc_channel_creation_or_update(
     };
 
     for trade in trades {
-        db::insert_trade(trade)?;
+        new_trade(trade)?;
```

Comment:
🙃 We should probably consider batch inserting the trades here as well.

Reply:
Here it probably makes sense because it's not that useful to insert 1 of the 2 trades when changing position direction.

In all other cases this should only be a single trade though.